packages/components/src/components/molecules/FiltersPreviewBoostingTabs/FiltersPreviewBoostingTabs.tsx

```diff
@@ -1,86 +1,141 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import CustomTabs from '../layout/tabs/CustomTabs'
 import TreeSelector from '../../atoms/form/TreeSelector'
-import { ITab, ITreeItem } from '@elastic-suite/gally-admin-shared'
-import { useTranslation } from 'react-i18next'
+import { IOptions, ITab, ITreeItem } from '@elastic-suite/gally-admin-shared'
+import { useTranslation } from 'next-i18next'
 import FiltersPreviewBoostingTab, {
   IContainerProps,
 } from './FiltersPreviewBoostingTab'
 import SearchBar from '../../atoms/form/SearchBar'
+import DropDownError from '../../atoms/form/DropDownError'
 export interface IPreviewBoostFilter {
   type: 'search' | 'category'
-  value: string | ITreeItem
+  search?: string
+  category?: string | number
+  localizedCatalog: string
 }
 
 interface IProps {
   onSendFilter: (filter: IPreviewBoostFilter) => void
   categories: ITreeItem[]
+  localizedCatalogsOptions: IOptions<string>
+  displayEmptyMessage: boolean
 }
 
 function FiltersPreviewBoostingTabs({
   onSendFilter,
   categories,
+  localizedCatalogsOptions,
+  displayEmptyMessage,
 }: IProps): JSX.Element {
+  const { t: tBoost } = useTranslation('boost')
+  const { t: tCommon } = useTranslation('common')
+
   const [category, setCategory] = useState<ITreeItem>(null)
   const [search, setSearch] = useState('')
-  const [displayEmptyMessage, setDisplayEmptyMessage] = useState(true)
+  const [localizedCatalog, setLocalizedCatalog] = useState('')
+  const [tabId, setTabId] = useState(0)
+  const [localizedCatalogError, setLocalizedCatalogError] = useState(false)
 
-  const { t } = useTranslation('boost')
-  const filterByTabId: IPreviewBoostFilter[] = [
-    { type: 'search', value: search },
-    { type: 'category', value: category },
-  ]
+  const sendFilter = (): void => {
+    onSendFilter({
+      type: tabId === 0 ? 'search' : 'category',
+      search,
+      category: category != null ? category.id : null,
+      localizedCatalog,
+    })
+  }
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(sendFilter, [onSendFilter, category, localizedCatalog, tabId])
+
+  const DrowDownLocalizedCatalog = (
+    <DropDownError
+      style={{ fontSize: '12px' }}
+      onChange={(value: string): void => {
+        setLocalizedCatalogError(Boolean(!value))
+        setLocalizedCatalog(value)
+        setCategory(null)
+      }}
+      value={localizedCatalog}
+      options={localizedCatalogsOptions}
+      placeholder={tCommon('localizedCatalog.placeholder')}
+      useGroups
+      sx={{ minWidth: '252.667px' }}
```

Comment:
`sx` et ` style` peuvent être combiné et il vaut mieux utiliser un styled component si possible par souci d'homogénéité.